packages/web/src/features/users/ForgotPasswordView.tsx

```diff
@@ -0,0 +1,68 @@
+import ModalView, { ModalViewTitle } from '../../shared/components/ModalView';
+import FormLabel from '../../shared/components/form/FormLabel';
+import InputError from '../../shared/components/form/InputError';
+import SubmitButton from '../../shared/components/form/SubmitButton';
+import TextInput from '../../shared/components/form/TextInput';
+import { useForm } from 'react-hook-form';
+import { useFlash } from '../../shared/hooks/flash';
+import apiClient from '../../shared/apiClient';
+import { ApiClientError } from '@translation/api-client';
+import Form from '../../shared/components/form/Form';
+import { useTranslation } from 'react-i18next';
+import useTitle from '../../shared/hooks/useTitle';
+
+export default function ForgotPasswordView() {
+  const { t } = useTranslation(['users']);
+  useTitle(t('users:forgot_password'));
+  const flash = useFlash();
+
+  const formContext = useForm<{ email: string }>();
+  async function onSubmit({ email }: { email: string }) {
+    try {
+      await apiClient.auth.forgotPassword({ email });
+      flash.success(t('users:reset_password_link_sent', { email }));
+    } catch (error) {
+      if (error instanceof ApiClientError && error.status === 401) {
+        flash.error(t('users:errors.invalid_auth'));
+      } else {
```

Comment:
We can remove this 401 error check now too since that error isn't possible anymore

Reply:
Done.